bin/git-forgit

```diff
@@ -602,18 +613,14 @@ _forgit_blame() {
         $FORGIT_FZF_DEFAULT_OPTS
         $FORGIT_BLAME_FZF_OPTS
     "
-    flags=$(git rev-parse --flags "$@")
-    preview="
-        if $_forgit_is_file_tracked; then
-            git blame {} --date=short $FORGIT_BLAME_GIT_OPTS $flags | $_forgit_blame_pager
-        else
-            echo File not tracked
-        fi
-    "
-    file=$(FZF_DEFAULT_OPTS="$opts" fzf --preview="$preview")
+    ${IFS+"false"} && unset old_IFS || old_IFS="$IFS"
+    IFS=$'\n'
+    #shellcheck disable=2207
+    flags=($(git rev-parse --flags "$@"))
+    ${old_IFS+"false"} && unset IFS || IFS="$old_IFS"
+    file=$(FZF_DEFAULT_OPTS="$opts" fzf --preview="$FORGIT blame_preview {} ${flags[*]}")
```

Comment:
I would propose moving this into a function, e.g. `_forgit_extract_flags`.
As this is a bugfix, though, I would transfer the change into a follow-up bugfix PR to keep this refactoring PR clean.

Reply:
This does make sense to me. I just looked into this and actually the `flags` variable has to be an array so that we can get rid of the `eval` in the last line. This is one of the reasons I made this change. I forgot to make that clear in the description. Because getting rid of `eval` usage is one of the goals of this PR and I think we should keep it as is and I will do a follow up PR that moves this into a `_forgit_extract_flags` function. Is that fine with you?

Reply:
I see, thanks for the explanation. I'm fine with that.